standalone-metastore/src/main/java/org/apache/hadoop/hive/metastore/messaging/event/filters/DatabaseAndTableFilter.java

```diff
@@ -43,7 +43,8 @@ public DatabaseAndTableFilter(final String databaseNameOrPattern, final String t
   private boolean isTxnRelatedEvent(final NotificationEvent event) {
     return ((event.getEventType().equals(MessageFactory.OPEN_TXN_EVENT)) ||
             (event.getEventType().equals(MessageFactory.COMMIT_TXN_EVENT)) ||
-            (event.getEventType().equals(MessageFactory.ABORT_TXN_EVENT))
+            (event.getEventType().equals(MessageFactory.ABORT_TXN_EVENT)) ||
+            (event.getEventType().equals(MessageFactory.ALLOC_WRITE_ID_EVENT))
```

Comment:
Allocate write ID is a DB/table related event and should be ignored if not lookup on this DB/table.